src/rulesets/oas/functions/__tests__/oasTagDefined.test.ts

```diff
@@ -17,7 +17,7 @@ describe('oasTagDefined', () => {
 
   test('validate a correct object', async () => {
     const results = await s.run({
-      swagger: 2.0,
+      swagger: '2.0',
```

Comment:
@philsturgeon Shouldn't we drop this (as I believe this rule applies to oas2 and oas3)? Or add a oas3 oriented test?

Reply:
Drop em!

Reply:
Dropped ;-)